### PR TITLE
Fix content type on the proxy response

### PR DESCRIPTION
**Pull request by gbiellem**
I think this should fix the content type not flowing through to the response.  See #5

### CODE REVIEW

**Comment by andrecarlucci** [comment]
Good catch, @gbiellem ! 
Would you be so nice and write an unit test for it please? 
We're trying to improve the coverage on every commit :)
Thanks!

**Comment by gbiellem** [comment]
@andrecarlucci No problem.  Test added :)

File: test/SharpReverseProxy.Tests/HttpContextFakes/FakeHttpMessageHandler.cs [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+using System.Net.Http.Headers;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace SharpReverseProxy.Tests.HttpContextFakes {
     public class FakeHttpMessageHandler : HttpMessageHandler {
         public HttpRequestMessage RequestMessage { get; private set; }
 
-        public HttpResponseMessage ResponseMessageToReturn { get; set; } = new HttpResponseMessage(HttpStatusCode.OK);
+        public HttpResponseMessage ResponseMessageToReturn { get; set; } = new HttpResponseMessage {
```
Comment (andrecarlucci) [inline test/SharpReverseProxy.Tests/HttpContextFakes/FakeHttpMessageHandler.cs]: Hi @gbiellem , 
This is a general purpose class for testing any request/response, even the ones with other content types. 
What do you think about moving this logic to the test itself?

Comment (gbiellem) [reply test/SharpReverseProxy.Tests/HttpContextFakes/FakeHttpMessageHandler.cs]: Makes sense.  I'll move it

**Comment by andrecarlucci** [comment]
Awesome, thanks!

**Comment by gbiellem** [comment]
Thanks @andrecarlucci - Love this project.  Looking forward to using it in a couple of places :)

**Pull request merged**